docs/user/customization.md

```diff
@@ -384,6 +384,67 @@ For example:
     worker-edab0895c59dba7a566f4b955d87d964                     3.11.0-744-g5b05d9d3-dirty   2.2.0             137m
     ```
 
+#### Nodes with Custom Kernel Arguments
+
+Custom kernel arguments can be applied to through manifests as an installer operation, and can also be applied as a [MachineConfig][machine-config] as a day 2 operation. The kernel arguments are applied upon boot and will be honored by the Machine-Config-Operator from then on.
+
+Example application of `loglevel=7` (change Linux kernel log level to KERN_DEBUG) for master nodes:
+
+1. Run `manifests` target to create all the manifests.
+
+    ```console
+    $ mkdir log_debug_cluster
+
+    $ openshift-install --dir log_debug_cluster create manifests
+    ...
+
+    $ ls -l log_debug_cluster/openshift
+    ...
+    99_openshift-machineconfig_99-master-ssh.yaml
+    99_openshift-machineconfig_99-worker-ssh.yaml
+    ...
+    ```
+
+2. . Create a `MachineConfig` that adds a kernel argument to change log level:
+
+    ```sh
+    cat > log_debug_cluster/openshift/99-master-kargs.yaml <<EOF
+    apiVersion: machineconfiguration.openshift.io/v1
+    kind: MachineConfig
+    metadata:
+     labels:
+       machineconfiguration.openshift.io/role: "master"
+     name: 99-master-kargs
+    spec:
+     kernelArguments:
```

Comment:
the spacing is wrong here.. ?

Reply:
some of this yaml is one space, well the other is 2 spaces?

Reply:
fixed